usaspending_api/api_contracts/contracts/subawards/Subawards.md

```diff
@@ -0,0 +1,52 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Subawards
+
+These endpoints are used to power USAspending.gov's subaward listings.
+
+# Subaward List
+
+This endpoint returns a filtered set of subawards.
+
+## Subawards [/api/v2/subawards/]
+
+This endpoint returns a list of data that is associated with the award profile page.
+
+### Subawards [POST]
+
++ Request (application/json)
+    + Attributes (object)
+        + page: 1 (required, number)
+        + limit: 10 (optional, number)
+        + filters (required, FilterObject)
+        + order: (required, enum)
+            + asc
+            + desc 
+            
++ Response 200 (application/json)
+    + Attributes
+        + results(required, array[SubawardResponse], fixed-type)
+        + page_metadata(required, PageMetadataObject)
+
+# Data Structures
+
+## SubawardResponse (object)
++ subaward_number (required, string)
```

Comment:
@tony-sappe So I don't have to go back on this, are we or aren't we doing backticks moving forward?

Reply:
No offense to @tony-sappe, it's not really his decision to make.  [It's already been made for him.](https://apiblueprint.org/documentation/mson/specification.html#6-reserved-characters--keywords)

```
6 Reserved Characters & Keywords

When using following characters or keywords in a Property Name, Literal Value or
Type Name the name or literal MUST be escaped in backticks `. Otherwise, a code
span MAY be used for any arbitrary formatting and has no specific meaning in an
MSON document.

6.1 Characters

:, (,), <, >, {, }, [, ], _, *, -, +, `
```

Reply:
Ever since underscores were determined to be problematic, the pattern has been to use backticks for 100% of example string values and keywords when they contain underscores.

I put a few notes in the README. https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts #6 deals with backticks of keywords